understanding/focus-appearance.html

```diff
@@ -53,389 +53,316 @@
+                     <p>上記の例のように、インジケータをフォーカスコンポーネントから少しずらすことは、達成基準の最小面積の要件を満たすために必要ではないが、インジケータをより見やすくするのに役立つ。CSS では、通例 <code>outline</code> 及び <code>outline-offset</code> プロパティがこれを達成するために使用される。
                      </p>
                      
                      
-                     <p>The smallest possible 2 CSS pixel thick indicator that is still a "perimeter" is a
-                        solid line that appears inside the component against the component's outer edge, for
-                        example by using a CSS <code>border</code> property. Indicators that are inset further within the component (not directly against
-                        the component's outer edge) need to be thicker than 2 CSS pixels to meet the minimum
-                        size requirement.
+                     <p>可能な限り小さい 2 CSS ピクセルの太さのインジケータで、「境界」であるものは、例えば CSS <code>border</code> プロパティを使うことで、コンポーネントの外辺に対してコンポーネントの内側に現れる実線である。(コンポーネントの外辺に直接ではなく) コンポーネント内にさらに挿入されるインジケータは、最小サイズの要件を満たすために 2 CSS ピクセルより太くする必要がある。
                      </p>
                      
                      
                      <figure id="focus-offsets">
-                        <img src="img/focus-indicator-offset-types.png" alt="a button along with 4 examples of different 2px solid focus indicators for it: offset outside the component, an outline around the boundary of the component, a border inside the boundary of the component, and inset inside the component" />
+                        <img src="img/focus-indicator-offset-types.png" alt="ボタンと、対応する四つの 2px の実線のフォーカスインジケータの例: コンポーネントの外側のオフセット、コンポーネントの境界の周りのアウトライン、コンポーネントの境界の内側のボーダー、コンポーネントの内側のインセット" />
                         
-                        <figcaption>Figure 4 All four of these example focus indicators are 2px solid lines. The "outset", "outline",
-                           and "border" indicators pass. The "inset" indicator does not meet the minimum area
-                           requirement and fails; it would need to be at least 3px thick to pass.
+                        <figcaption>図 4 四つフォーカスインジケータのこれら例は全て 2px の実線である。"outset"、"outline"及び"border" のインジケータは合格する。"inset" のインジケータは最小面積の要件を満たさないため不合格となる。
                         </figcaption>
                         
                      </figure>
                      
                      
-                     <p>Note that different Non-text Contrast requirements may apply depending on whether
-                        the focus indicator is offset from, inset into, or against the edge of the component.
-                        See the <a href="#relationship-with-non-text-contrast">Relationship with Non-text Contrast</a> section below.
+                     <p>フォーカスインジケータがコンポーネントの端からオフセットされている、はめ込まれている、又は辺に当たっているかによって、異なる非テキストコントラスト要件が適用される可能性があることに注意する。下記の<a href="#relationship-with-non-text-contrast">非テキストコントラストとの関係</a>を参照。
```

Comment:
「非テキストコントラスト」は、達成基準名に合わせて「非テキストのコントラスト」としました。

```suggestion
                     <p>フォーカスインジケータがコンポーネントの端からオフセットされている、はめ込まれている、又は辺に当たっているかによって、異なる非テキストコントラスト要件が適用される可能性があることに注意する。下記の<a href="#relationship-with-non-text-contrast">非テキストのコントラストとの関係</a>を参照。
```